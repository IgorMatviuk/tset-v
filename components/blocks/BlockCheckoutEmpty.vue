<template>
  <div class="checkout-empty">
    <BContainer>
      <div class="checkout-empty__text">{{ $t('cart.empty') }}</div>
      <BButton to="/" variant="primary">{{ $t('cart.continue') }}</BButton>
    </BContainer>
  </div>
</template>

<script>
// TODO: empty cart layout
export default {}
</script>

<style lang="scss">
.checkout-empty {
  &__text {
    max-width: 305px;
    margin-bottom: 22px;
    font-size: 14px;
    color: $text-color;
  }

  @include media-breakpoint-up(md) {
    &__text {
      max-width: 620px;
    }
  }

  @include media-breakpoint-up(xl) {
    &__text {
      max-width: 470px;
    }
  }
}
</style>
