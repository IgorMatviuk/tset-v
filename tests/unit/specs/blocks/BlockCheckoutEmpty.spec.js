import { mount, createLocalVue } from '@vue/test-utils'

import { BContainer, BButton } from 'bootstrap-vue'

import BlockCheckoutEmpty from '@/components/blocks/BlockCheckoutEmpty'

const localVue = createLocalVue()
localVue.component('BContainer', BContainer)
localVue.component('BButton', BButton)

describe('BlockPostNavigation', () => {
  function createBlockCheckoutEmpty() {
    const wrapper = mount(BlockCheckoutEmpty, {
      localVue,
      mocks: {
        $t: (path) => path
      }
    })

    return wrapper
  }

  test('render privacy policy block', () => {
    const wrapper = createBlockCheckoutEmpty()

    expect(wrapper.element).toMatchSnapshot()
  })
})
