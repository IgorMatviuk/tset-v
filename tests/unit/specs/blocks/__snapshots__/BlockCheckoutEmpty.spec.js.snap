// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockPostNavigation render privacy policy block 1`] = `
<div
  class="checkout-empty"
>
  <div
    class="container"
  >
    <div
      class="checkout-empty__text"
    >
      cart.empty
    </div>
     
    <a
      class="btn btn-primary"
      href="/"
      target="_self"
    >
      cart.continue
    </a>
  </div>
</div>
`;
